app/scripts/proactive/view/TaskView.js

```diff
@@ -285,6 +277,8 @@ define(
 
             if (!model) return;
 
+            model.model.parentModel = this.model;
```

Comment:
model.model.parentModel: there are a lot of models, is it normal to put a model in the model?

Reply:
I can work to make this code more understandable, yes.